### ISSUE TITLE
Integration with AWS

### ISSUE THREAD

**Issue by wlandau-lilly** [mllg/batchtools#151]
Would it make sense for `batchtools` to talk to AWS?

**Comment by mllg** [comment]
Yes, definitely! I already started to play around with AWS but did not find the time yet to implement it. I was able to use the SSH mode on some AWS nodes though, and @berndbischl was performed some computations on Azure in this mode.

**Comment by berndbischl** [comment]
> , and @berndbischl was performed some computations on Azure in this mode.

that was some time ago, but worked very well. (i also had to use sshfs)

i do think that AWS has become so important, that maybe bt could integrate it more "officially" but i dont know how large the time investment for michel would be....
